tests/test_dynamodb.py

```diff
@@ -103,7 +107,16 @@ def test_dynamodb_mark(self):
         self.assertEqual(len(resources), 1)
         tags = client.list_tags_of_resource(ResourceArn=arn)
         tag_map = {t["Key"]: t["Value"] for t in tags["Tags"]}
-        self.assertTrue("test_key" in tag_map)
+
+        localtz = zoneinfo.gettz("America/New_York")
```

Comment:
I don't really like the code here to validate the date that was posted on the tag, I've looked at [freezegun](https://github.com/spulec/freezegun) to make this more intuitive, thoughts?

Reply:
its a little unclear to me that freeze gun is going to do anything particularly useful here, time is already frozen in the recorded files. the rest of this is just constructing a comparison value which afaics freeze gun doesn't help with.

generally trying to avoid extra dependencies unless their adding significant value, fwiw we have a variation on freezing time in the off hour tests.

Reply:
I just went through the code on freezegun, as far as implementations and dependencies go, it looks pretty sane and lightweight.